src/silo/database.cpp

```diff
@@ -105,20 +108,15 @@ void Database::saveDatabaseState(const std::filesystem::path& save_directory) {
 
    SPDLOG_INFO("Saving database schema");
 
-   auto yaml_string =
-      YAML::Dump(schema::DatabaseSchema{{{schema::TableName::getDefault(), table->schema}}}.toYAML()
-      );
-   const auto database_schema_filename = versioned_save_directory / "database_schema.yaml";
-   std::ofstream database_schema_file{database_schema_filename};
-   database_schema_file << yaml_string;
-   database_schema_file.close();
+   const auto database_schema_path = versioned_save_directory / DATABASE_SCHEMA_FILENAME;
```

Comment:
so previously we were storing a yamlstring of the table->schema and now we are storing a binary_oarchive of tables?